app/components/Resources/ResourcesTable.js

```diff
@@ -65,7 +67,7 @@ class ResourcesTable extends Component {
 		let offset = page ? 1 : 0;
```

Comment:
I think it was part of an attempt to quickly fix something that I since fixed for real. removed.